chapter_optimization/adadelta.md

```diff
@@ -83,6 +83,28 @@ def adadelta(params, states, hyperparams):
         p.grad.data.zero_()
 ```
 
+```{.python .input}
+#@tab tensorflow
+%matplotlib inline
+from d2l import tensorflow as d2l
+import tensorflow as tf
+
+def init_adadelta_states(feature_dim):
+    s_w = tf.Variable(d2l.zeros((feature_dim, 1)))
+    s_b = tf.Variable(d2l.zeros(1))
+    delta_w = tf.Variable(d2l.zeros((feature_dim, 1)))
+    delta_b = tf.Variable(d2l.zeros(1))
+    return ((s_w, delta_w), (s_b, delta_b))
```

Comment:
Remove redundant parenthesis

Reply:
@terrytangyuan Are you talking about this ```((s_w, delta_w), (s_b, delta_b))``` parenthesis. If so then this parenthesis is used in both mxnet and pytorch.